de.gunibert.Hackgregator.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "de.gunibert.Hackgregator",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "hackgregator",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--buildtype=release"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://source.puri.sm/Librem5/libhandy/-/archive/v0.0.10/libhandy-v0.0.10.tar.gz",
+                    "sha256" : "9f4528c07bdb9796300ff0efb165cdc48cfcd7f28952c9d776de15b735844981"
+                }
+            ]
+        },
+        {
+            "name": "hackernews-glib",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/gunibert/hackernews-glib.git",
+                    "tag" : "0.2.0"
```

Comment:
A commit is needed here & for a hackgreator too.

Reply:
I think this has already a tag. Or what do you mean by that?